src/common/pcqueue.cc

```diff
@@ -28,6 +28,7 @@
 
 #include "common/massert.h"
 #include "devtools/TracePrinter.h"
+#include "sfserr/sfserr.h"
```

Comment:
This will work because the file is in the same directory than the sfserr folder. But in other places the include was like: "common/sfserr/sfserr.h". Consider changing it to maintain consistency (minor suggestion).